perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -112,13 +115,24 @@ then
  python3 ./perfmetrics/scripts/presubmit/print_results.py
 fi
 
-# Execute integration tests.
-if [[ "$integrationTestsStr" == *"$EXECUTE_INTEGRATION_TEST_LABEL"* ]];
+# Execute integration tests on zonal bucket(s).
+if test -n "${integrationTestsOnZBStr}" ;
 then
   echo checkout PR branch
   git checkout pr/$KOKORO_GITHUB_PULL_REQUEST_NUMBER
 
-  echo "Running e2e tests...."
+  echo "Running e2e tests on zonal bucket(s) ..."
+  # $1 argument is refering to value of testInstalledPackage.
+  ./tools/integration_tests/run_e2e_tests.sh $RUN_E2E_TESTS_ON_INSTALLED_PACKAGE $SKIP_NON_ESSENTIAL_TESTS_ON_PACKAGE $BUCKET_LOCATION $RUN_TEST_ON_TPC_ENDPOINT $RUN_TESTS_WITH_PRESUBMIT_FLAG true
+fi
+
+# Execute integration tests on non-zonal bucket(s).
+if test -n "${integrationTestsStr}" ;
+then
+  echo checkout PR branch
+  git checkout pr/$KOKORO_GITHUB_PULL_REQUEST_NUMBER
+
+  echo "Running e2e tests on non-zonal bucket(s) ..."
   # $1 argument is refering to value of testInstalledPackage.
   ./tools/integration_tests/run_e2e_tests.sh $RUN_E2E_TESTS_ON_INSTALLED_PACKAGE $SKIP_NON_ESSENTIAL_TESTS_ON_PACKAGE $BUCKET_LOCATION $RUN_TEST_ON_TPC_ENDPOINT $RUN_TESTS_WITH_PRESUBMIT_FLAG ${RUN_TESTS_FOR_ZONAL_BUCKET}
```

Comment:
should this be 
```suggestion
  ./tools/integration_tests/run_e2e_tests.sh $RUN_E2E_TESTS_ON_INSTALLED_PACKAGE $SKIP_NON_ESSENTIAL_TESTS_ON_PACKAGE $BUCKET_LOCATION $RUN_TEST_ON_TPC_ENDPOINT $RUN_TESTS_WITH_PRESUBMIT_FLAG false
```
or we can use this flag for both cases by setting it appropriately at the start so that it is more readable..

Reply:
yes, this is a miss because of the rebased caused by the fix made in the previous PR on the same line.

Reply:
fixed now.

Reply:
Shall we remove unused RUN_TESTS_FOR_ZONAL_BUCKET variable too?

Reply:
done